pkg/controllers/localbuild/nginx.go

```diff
@@ -0,0 +1,134 @@
+package localbuild
```

Comment:
We need tests for these.

Reply:
darn, you noticed!

Reply:
We can punt tests for now imo. We should work on getting gitea working first. 
Just need to open an issue and it can be a good first time issue for new contributors.